lib/controllers/include/Motor.h

```diff
@@ -1,8 +1,7 @@
 #pragma once
 
-#ifndef MOTOR_DEFAULT_FREQUENCY_HZ
-#define MOTOR_DEFAULT_FREQUENCY_HZ 244  // 10 kHz
-#endif
+#include <bits/stdint-uintn.h>
```

Comment:
never include the bits header!

the bits header is an internal header created by gcc. It includes literally everything in the standard library and a lot of other crap. It also isnt guaranteed to work because its just gcc's internal implementation. The proper portable way to do this is to always include the proper header. 

In this case it looks like you're trying to include fixed width integers. In C++, they exist at `<cstdint>`.